react_paragraphs.permissions.yml

```diff
@@ -0,0 +1,31 @@
+add paragraphs row entities:
+  title: 'Create new Paragraphs Row entities'
+
+administer paragraphs row entities:
+  title: 'Administer Paragraphs Row entities'
+  description: 'Allow to access the administration form to configure Paragraphs Row entities.'
+  restrict access: true
+
+delete paragraphs row entities:
+  title: 'Delete Paragraphs Row entities'
+
+edit paragraphs row entities:
+  title: 'Edit Paragraphs Row entities'
+
+view published paragraphs row entities:
+  title: 'View published Paragraphs Row entities'
+
+view unpublished paragraphs row entities:
+  title: 'View unpublished Paragraphs Row entities'
```

Comment:
Are you able to unpublish a row?

Reply:
not in the UI at this time.

Reply:
I've removed these permissions from the file since they are currently unused